test/e2e/apimachinery/flowcontrol.go

```diff
@@ -150,28 +154,31 @@ var _ = SIGDescribe("API priority and fairness", func() {
 	// the two clients and not allow one client to drown out the other despite
 	// having a higher QPS.
 	ginkgo.It("should ensure that requests can't be drowned out (fairness)", func() {
-		priorityLevelName := "e2e-testing-prioritylevel"
-		flowSchemaName := "e2e-testing-flowschema"
+		priorityLevelName := "e2e-testing-prioritylevel-" + f.UniqueName
+		flowSchemaName := "e2e-testing-flowschema-" + f.UniqueName
 		loadDuration := 10 * time.Second
 
 		framework.Logf("creating PriorityLevel %q", priorityLevelName)
 		_, cleanup := createPriorityLevel(f, priorityLevelName, 1)
 		defer cleanup()
 
+		highQPSClientName := "highqps-" + f.UniqueName
+		lowQPSClientName := "lowqps-" + f.UniqueName
 		framework.Logf("creating FlowSchema %q", flowSchemaName)
-		_, cleanup = createFlowSchema(f, flowSchemaName, 1000, priorityLevelName, "*")
+		_, cleanup = createFlowSchema(f, flowSchemaName, 1000, priorityLevelName, []string{highQPSClientName, lowQPSClientName})
 		defer cleanup()
 
 		type client struct {
-			username              string
-			qps                   float64
-			concurrencyMultiplier float64
-			concurrency           int32
-			completedRequests     int32
+			username                    string
+			qps                         float64
+			concurrencyMultiplier       float64
+			concurrency                 int32
+			completedRequests           int32
+			expectedCompletedPercentage float64
 		}
 		clients := []client{
-			{username: "highqps", qps: 100.0, concurrencyMultiplier: 2.0},
-			{username: "lowqps", qps: 5.0, concurrencyMultiplier: 0.5},
+			{username: highQPSClientName, qps: 90, concurrencyMultiplier: 2.0, expectedCompletedPercentage: 0.90},
+			{username: lowQPSClientName, qps: 4, concurrencyMultiplier: 0.5, expectedCompletedPercentage: 0.90},
```

Comment:
Isn't this .9 an increase over #96798's .75?

Reply:
A recent failure:

`Dec  4 15:38:28.824: client "highqps-apf-8126": got 36.6% completed requests, want at least 75.0%`

So I think we'll need a followup.

Perhaps we need to use quite large share amounts to throttle the rest of the cluster.